### PR TITLE
Initial commits

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description ##

This pull request contains the initial commits for this repository.

## 💭 Motivation and context ##

We require an AMI like this for cisagov/cool-system#204, for example.

## 🧪 Testing ##

All `pre-commit` hooks pass.  I also used these changes to create an AMI for our COOL staging environment, and I verified that the `vnc` user was able to run `terraform` and `terraform-docs`.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - _eschew scope creep!_
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated to reflect the changes in this PR.
- [x] All new and existing tests pass.
- [x] Build and test a new staging AMI with these changes.

## ✅ Pre-merge checklist ##

- [x] Revert to default branch of `terraform` Ansible role after cisagov/ansible-role-terraform#2 is approved and merged.
- [x] Revert to default branch of `assessment_tool` Ansible role after cisagov/ansible-role-assessment-tool#16 is approved and merged.
- [x] Finalize version.
- [ ] Build new production AMI with these changes.

### CODE REVIEW

File: src/packer.json [hunk 1]
```diff
@@ -46,7 +46,7 @@
         }
       },
       "tags": {
-        "Application": "Example",
+        "Application": "COOL",
```
Comment (dav3r) [inline src/packer.json]: We usually make the Application tag specific to the AMI:
* https://github.com/cisagov/assessor-portal-packer/blob/b8771840660e1fbb037f635e509e1541da580361/src/packer.json#L49
* https://github.com/cisagov/kali-packer/blob/1cd91902ad6f92146f67669c8b31d42ec3236f03/src/packer.json#L49
* https://github.com/cisagov/nessus-packer/blob/bb95dc457d11d410052ca477dbe208f4439c87b9/src/packer.json#L49
* https://github.com/cisagov/pca-gophish-composition-packer/blob/c1e30c7ec3aa5a4c9ad391f31f815644dd948ebe/src/packer.json#L49
```suggestion
        "Application": "Terraformer",
```

File: src/terraform.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    - role: terraform
+      vars:
+        # According to the RTA folks they are on 0.14.  This is the
+        # most recent 0.14 version.
+        terraform_version: 0.14.11
+    - role: assessment_tool
+      vars:
+        group: vnc
+        install_dir: /home/vnc/
+        owner: vnc
```
Comment (dav3r) [inline src/terraform.yml]: Don't assume that VNC user's username is always going to be`vnc`, even though that is the [default in cisagov/ansible-role-vnc-server](https://github.com/cisagov/ansible-role-vnc-server/blob/9f7886e7067fddb069344d814e304ad06345f67f/defaults/main.yml#L8-L9).  You should grab the username from SSM the same way as in the [VNC playbook](https://github.com/cisagov/terraformer-packer/blob/fd7edf26bd44705b16b251c71583a4343ca4cd21/src/vnc.yml#L98).

File: src/vnc.yml [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+        create: yes
+        group: "{{ username }}"
+        line: "{{ item }}"
+        mode: 0644
+        owner: "{{ username }}"
+        path: /home/{{ username }}/.bashrc
+      loop:
+        - export EFS_SHARE=/share
+        - export PCA_OPS_PATH=${EFS_SHARE}/PCA
+        - export PCA_DEV_PATH=${EFS_SHARE}/private
```
Comment (dav3r) [inline src/vnc.yml]: I don't think this block is needed in this AMI.
```suggestion
```

**Review by dav3r — CHANGES_REQUESTED** [review]
Looks great, but I see some things that should be taken care of.

File: src/terraform.yml (hunk 2 above)
Comment (jsf9k) [reply src/terraform.yml]: Great catch!  See commit cadd83b.

**Review by dav3r — APPROVED** [review]
🎵 Terraformers, more than meets the eye.
Terraformers, infrastructure in disguise! 🎵

File: src/cloud_init.yml [hunk 4]
```diff
@@ -0,0 +1,18 @@
+---
+# For some reason cloud-init-local, cloud-config, and cloud-final are
+# not enabled in the base Kali AMI.  We need cloud-config enabled, and
```
Comment (mcdonnnj) [inline src/cloud_init.yml]: This image is built on Debian Buster but this comment references Kali.

File: src/ufw.yml [hunk 5]
```diff
@@ -0,0 +1,33 @@
+---
+- hosts: all
+  name: Install ufw
```
Comment (mcdonnnj) [inline src/ufw.yml]: Consistency.

```suggestion
  name: Install UFW
```

**Review by mcdonnnj — CHANGES_REQUESTED** [review]
Some great work here. I did have two small things I would like to see fixed.

File: src/cloud_init.yml (hunk 4 above)
Comment (dav3r) [reply src/cloud_init.yml]: @jsf9k For reference, see [here](https://github.com/cisagov/assessor-portal-packer/blob/e7c1b2dc6aac91c1e9a0ccdca0eae2c7c0ac4bcc/src/cloud_init.yml#L2-L3) for how I modified that comment in assessor-portal-packer.

Comment (jsf9k) [reply src/cloud_init.yml]: See commit e8722fb.

**Review by mcdonnnj — APPROVED** [review]
LGTM ✔

**Pull request merged**